google/resource_google_project.go

```diff
@@ -103,6 +103,7 @@ func resourceGoogleProject() *schema.Resource {
 
 func appEngineResource() *schema.Resource {
 	return &schema.Resource{
+		CustomizeDiff: resourceGoogleProjectAppEngineCustomizeDiff,
 		Schema: map[string]*schema.Schema{
 			"name": &schema.Schema{
 				Type:     schema.TypeString,
```

Comment:
Not actually related to this change, but can you order the properties in Required -> Optional -> Computed order? I'm a big fan of that to make it easy to immediately see which fields are settable.

Reply:
👍